### PR TITLE
fix: set background colour for tooltips & inline code blocks

### LINKED ISSUE #44: popin do not have backgrounds
Example for time but it also does it on every other popins (labels, contribution matrix, ...)

image

### PR DESCRIPTION
**Pull request by S0nter**
With release [1.0.0](https://github.com/catppuccin/gitea/releases/tag/v1.0.0) and gitea version v1.22.3 inline codes (`this`) don't have a background and popups'/tooltip's background is transparent.

Here is a comparison with Catppuccin Frappé theme applied

### Before:
inline1
tooltip1

### After:
inline2
tooltip2

Fixes #44

### CODE REVIEW

**Comment by sgoudham** [comment]
Hey :wave:,
Thanks for the PR! Could you please include before/after screenshots, happy to merge after that!

**Comment by S0nter** [comment]
> Hey 👋, Thanks for the PR! Could you please include before/after screenshots, happy to merge after that!

Done.

**Comment by sgoudham** [comment]
Thanks for the screenshots, I made a change to the tooltip so that it's more readable when the background is also `base`:

image
image

**Review by sgoudham — APPROVED** [review]

**Pull request merged**
